db/migrate/20170430050223_backfill_library_timestamps.rb

```diff
@@ -0,0 +1,11 @@
+class BackfillLibraryTimestamps < ActiveRecord::Migration
+
+  def change
+    LibraryEntry.where("progress > 0").find_in_batches do |batch|
```

Comment:
I _think_ it does, it can access itself(self) by just doing something like `.update_in_batches(consumed_at: updated_at)` (but I could be wrong)